node-fs-writefile/cp.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+let dataOutput = '';
+
+fs.readFile(process.argv[2], 'utf8', (err, data) => {
+  if (err) {
+    console.error(err);
+    process.exit(1);
+  }
+  dataOutput = dataOutput + data + '\n';
+});
+
+fs.writeFile(process.argv[3], dataOutput, 'utf8', err => {
+  if (err) {
+    console.error(err);
+    process.exit(1);
+  }
+});
```

Comment:
You'll need to move this code into the callback of the readFile, since that's the only point at which we know the data has been read